packages/frontend/src/state/lp/hooks.ts

```diff
@@ -68,6 +69,62 @@ export const useOpenPositionDeposit = () => {
   return openPositionDeposit
 }
 
+// Close position with flashloan
+export const useClosePosition = () => {
```

Comment:
maybe naming could be improved (can have close positions w/o flashloans, etc)

Reply:
flashClosePosition?